labelbox/schema/project.py

```diff
@@ -352,6 +353,68 @@ def create_prediction(self, label, data_row, prediction_model=None):
         res = self.client.execute(query_str, params)
         return Prediction(self.client, res["createPrediction"])
 
+    def upload_annotations(
+        self,
+        name: str,
+        annotations: Union[str, Iterable[dict]],
+        validate_file: bool = True,
```

Comment:
Nit: having a flag that's applicable to only one type of arg isn't ideal, can we avoid it?